chapter_preliminaries/probability.md

```diff
@@ -1,304 +1,750 @@
+how to update the initial *prior* P(H)
+in light of the available evidence E
+to produce *posterior* beliefs 
+$P(H|E) = \frac{P(E|H) P(H)}{P(E)}$.
+Informally, this can be stated as 
+"posterior equals prior times likelihood, divided by the evidence".
+Now, because the evidence $P(E)$ is the same for all hypotheses,
+we can get away with using the simplified form
+and simply normalize over the hypotheses.
+
+Independence is another fundamentally important concept
+that you forms the backbone of 
+many important ideas in statistics.
+In short, two variables are *independent*
+if conditioning on the value of $A$ does not
+cause any change to the probability distribution
+associated with $B$ and vice versa.
+More formally, independence, denoted $A \perp B$, 
+requires that $P(A|B) = P(A)$ and, consequently, 
+that $P(A,B) = P(A|B) P(B) = P(A) P(B)$.
+Independence is often an appropriate assumption.
+For example, if the random variable A 
+represents the outcome from tossing one fair coin 
+and the random variable B 
+represents the outcome from tossing another,
+then knowing whether A came up heads
+should not influence the probability
+of B coming up heads.
+
+
+Independence is especially when it holds among the successive 
+draws of our data from some underlying distribution 
+(allowing us to make strong statistical conclusions)
+or when it holds among various variables in our data,
+allowing us to work with simpler models
+that encode this independence structure.
+On the other hand, estimating the dependencies 
+among random variables is often the very aim of learning.
+We care to estimate the probability of disease given symptoms
+specifically because we believe 
+that diseases and symptoms are *not* independent. 
+
+
+Note that because conditional probabilities are fully functioning probabilities,
+independence and dependence are properties that also apply there. 
+Two random variables $A$ and $B$ are conditionally independent 
+given a third variable $C$ if and only if $P(A, B|C) = P(A|C)P(B|C)$.
+Interestingly, two variables can be independent in general
+but become dependent when conditioning on a third. 
+This often occurs when the two random variables $A$ and $B$
+correspond to causes of some third variable $C$.
+For example, broken bones and lung cancer might be independent 
+in the general population but if we condition on being in the hospital
+then we might find that broken bones are negatively correlated with lung cancer. 
+That's because the broken bone *explains away* why some person is in the hospital
+and thus lowers the probability that they have lung cancer. 
+
+
+And conversely, two dependent random variables 
+can become independent upon conditioning on a third. 
+This often happens when two otherwise unrelated events
+have a common cause. 
+Shoe size and reading level are highly correlated 
+among elementary school students,
+but this correlation disappears if we condition on age. 
```

Comment:
"... but this correlation disappears if we condition on age"

This may not be true. E.g., the age is an elementary school age. Why not "... but this correlation disappears if we condition on the place of living"?

Reply:
Sure.